src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Ah, that's actually what I'm doing right now (minus showing in VS).  So you're saying not to do this as a build-time task, but as a run-on-command target?